### PR TITLE
Bluejay eeprom 205

### PR DESCRIPTION
**Pull request by mathiasvr**
Initial work on adding melody on beacon (5), and synchronising melody endings.

Based on @saidinesh5 work in https://github.com/mathiasvr/bluejay-configurator/pull/19.

### CODE REVIEW

File: src/Containers/App/index.jsx [hunk 1]
```diff
@@ -967,8 +967,14 @@ class App extends Component {
...
     const individual = [ ...escs.individual ];
     const converted = melodies.map((melody) => Rtttl.toBluejayStartupMelody(melody));
+
+    // Set wait time after melody to synchronize playback on all ESCs
+    const melodyDurations = melodies.map((melody) => Rtttl.parse(melody).melody.reduce((a, b) => a + b.duration, 0));
+    const maxMelodyDuration = Math.max(...melodyDurations);
+
     for(let i = 0; i < converted.length; i += 1) {
       individual[i].individualSettings.STARTUP_MELODY = converted[i].data;
+      individual[i].individualSettings.STARTUP_MELODY_WAIT_MS = maxMelodyDuration - melodyDurations[i];
```
Comment (saidinesh5) [inline src/Containers/App/index.jsx:977]: You'd want to cast an integer here? Default js numbers are floats no?

Comment (mathiasvr) [reply src/Containers/App/index.jsx:977]: It should be handled in the `objectToSettingsArray` function.

**Comment by mathiasvr** [comment]
@stylesuxx Is there a way to use this for a bluejay test version or is it easier to merge it like this and update later if necessary?

**Comment by stylesuxx** [comment]
> @stylesuxx Is there a way to use this for a bluejay test version or is it easier to merge it like this and update later if necessary?

Not entirely sure what you mean by test version? You should be able to flash your new image from disk and then test locally, no?

**Comment by mathiasvr** [comment]
@stylesuxx Oh yes I mean a public test release of bluejay. I think it might be easiest to release a version of esc configurator with these changes. The only thing is that this might not be the final version of eeprom 205 but it can probably just be updated later in that case.

**Comment by stylesuxx** [comment]
Yeah, I think that's the best option, I'll just merge the changes and we go from there.

**Pull request merged**